lib/widgets/task_widget.dart

```diff
@@ -34,7 +41,59 @@ class _TaskWidgetState extends State<TaskWidget> {
     }
 
     var tileContent;
-    if (widget.task.imageId == null) {
+    // if imageId is null, then this is not an image task
+    if (widget.task.imageId != null) {
+      // this is an image task
+      tileContent = [
+        RaisedButton(
+            onPressed: () async {
+              var id = widget.task.imageId;
+              Image img = await _fetchImage(id);
+              // route to image label page and wait for return
+              // return value will be the designated label
+              String selectedLabel = await Navigator.push(
+                context,
+                MaterialPageRoute(builder: (context) => FotoLabelPage(img)),
+              );
+
+              // if user has set a label, then save that label to the database
+              if (selectedLabel != null) {
+                try {
+                  BoxCommunicator().setLabel(id, selectedLabel);
+                  _deleteTask(widget.task);
+                } catch (e) {
+                  print(e);
+                }
+              }
+            },
+            child: Text(widget.task.description)),
+      ];
+    } else if (widget.task.title.compareTo("Baumkronen Foto") == 0) {
```

Comment:
Modularitaet: In weiteren User-stories sollte hier ein Interface bereitgestellt werden, um zu ueberpruefen, ob ein "Foto-mach" task vorliegt.